### PR TITLE
[ES] Unfuzzy all es.po, fixed a few errors, fixed/capitalized several terms.

### PR DESCRIPTION
**Pull request by retters**
This time I made sure the digital version has 53 pages, and the printable one has 56.

### CODE REVIEW

File: translations/combat.tex/es.po [hunk 1]
```diff
@@ -359,19 +295,16 @@ msgstr ""
...
 "  \\includegraphics[width=0.55\\linewidth]{\\art/wyvern.png}\n"
 "\\end{center}\n"
 "\n"
 msgstr ""
-"{\\centering\\phantom{Eje Estelar}\\par}\n"
-"\\includegraphics[width=\\linewidth]{\\art/unicorn.jpg}\n"
+"\\begin{center}\n"
+"  \\includegraphics[width=0.55\\linewidth]{\\art/wyvern.png}\n"
+"\\end{center}\n"
 "\n"
```
Comment (qwrtln) [inline translations/combat.tex/es.po:308]:
```suggestion
```
If there's nothing to translate, there's no need to write anything. Original string will be used.

File: translations/deckbuilding.tex/es.po [hunk 2]
```diff
@@ -111,19 +111,20 @@ msgstr ""
...
 msgid ""
 "\\vfill\n"
 "\\hfill\\includegraphics[width=0.9\\linewidth]{\\art/coin.png}\n"
 "\n"
 msgstr ""
-"{\\centering\\phantom{Eje Estelar}\\par}\n"
-"\\includegraphics[width=\\linewidth]{\\art/unicorn.jpg}\n"
+"\\vfill\n"
+"\\hfill\\includegraphics[width=0.9\\linewidth]{\\art/coin.png}\n"
 "\n"
```
Comment (qwrtln) [inline translations/deckbuilding.tex/es.po:128]:
```suggestion
```

File: translations/round_structure.tex/es.po [hunk 3]
```diff
@@ -117,3 +117,8 @@ msgid ""
 "}\n"
 "\n"
 msgstr ""
+"\\note{6}{\n"
+"  En juego de Alianza, los equipos toman turnos alternados.\n"
+"  Al inicio del primer turno de tu equipo en la Ronda, elije quién jugará primero en esa Ronda.\n"
+"}\n"
+"\n"
```
Comment (qwrtln) [inline translations/round_structure.tex/es.po:124]: Careful with the final newline, the check will fail.

**Comment by qwrtln** [comment]
Page count is correct now, good job 👍🏻 Please fix the other issues, and we're good to go.

**Comment by qwrtln** [comment]
> po4a

So close ;-) Try it in your terminal

**Review by qwrtln — APPROVED** [review]

**Review by luischos — APPROVED** [review]
nice job!

**Pull request merged**